### PR TITLE
Fixup `network.*` runner commands

### PR DESCRIPTION
**Pull request by mgfritch**
Fixes: https://bugzilla.suse.com/show_bug.cgi?id=1204438

### CODE REVIEW

File: xml/bp_troubleshooting_tips.xml [hunk 1]
```diff
@@ -422,10 +422,18 @@
    verifying expectations.
   </para>
 
+  <tip>
+   <title>Sync Runners</title>
+   <para>
+    Use the <command>salt-run saltutil.sync_runners</command> command
+    if the salt runner is reported as <literal>not available</literal>.
```
Comment (tbazant) [inline xml/bp_troubleshooting_tips.xml:429]:
```suggestion
    if the &salt; runner is reported as <literal>not available</literal>.
```

**Review by tbazant — APPROVED** [review]
one formal nit, otherwise OK

File: xml/bp_troubleshooting_tips.xml [hunk 2]
```diff
@@ -422,10 +422,18 @@
...
+  </tip>
+
   <sect2 xml:id="network-test-basic-diagnostics">
    <title>Performing basic diagnostics</title>
    <para>
-    Try the <command>salt '*' network.ping</command> command to ping between
+    Try the <command>salt-run network.ping</command> command to ping between
     cluster nodes to see if individual interface can reach to specific
     interface and the average response time. Any specific response time much
     slower then average will also be reported. For example:
```
Comment (dariavladykina) [inline xml/bp_troubleshooting_tips.xml:439]:
```suggestion
    cluster nodes to see if an individual interface can reach to a specific
    interface and the average response time. Any specific response time much
    slower than average will also be reported. For example:
```

**Review by dariavladykina — COMMENTED** [review]
Hi, a small suggestion here. Thanks!

**Pull request merged**